go.mod

```diff
@@ -20,11 +18,11 @@ require (
 	github.com/vishvananda/netlink v1.1.0
 	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae // indirect
 	golang.org/x/net v0.14.0
-	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.27.3
-	k8s.io/apimachinery v0.27.3
-	k8s.io/code-generator v0.27.2
-	k8s.io/component-base v0.27.2
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
don't indirect deps have its own block?

Reply:
Apparently when a dependency change from direct to indirect it doesn't change sections.
I removed that line and re-ran `go mod tidy` to clean this up.